src/lib/actions.js

```diff
@@ -60,10 +60,11 @@ export const requestRunLighthouse = store.action((state, url) => {
   })();
 
   return p.catch((err) => {
-    console.warn('failed to run Lighthouse', url, err);
+    const errMsg = err.name === 'FetchError' ? err.name : err.toString();
+    console.warn('failed to run Lighthouse', url, errMsg);
 
     const update = {
-      lighthouseError: err.toString(),
+      lighthouseError: errMsg,
```

Comment:
is lighthouseError every logged? It seems like errMsg still does err.toString() in some instances so I'm not sure I understand how this fixes the issue?

Reply:
maybe a better question:

What was causing the issue in the first place? Are only FetchErrors displaying the key or do other kinds of errors?

Reply:
LighthouseError is actually displayed in the UI. In case of FetchError, it means there was a problem in fetch from Pagespeed API, and the url (including key) of Pagespeed api is included in the error message. This is not the case for other errors which originate from Ligthouse service (and not pagespeed).